README_REACT.md

```diff
@@ -0,0 +1,4 @@
+# Running the app requires Webpack
+
+`foreman start -f Procfile.dev`
```

Comment:
We should consolidate this section in the main README.md. Replace the `rails s` part.

Reply:
Noted. I've been deferring the documentation a bit until we stabilise with Circle.CI + Heroku, as I foresee changes needed to get those 2 to run.